spec/musicalbum_spec.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../musicalbum'
+require_relative '../genre'
+
+describe '#constructor' do
+  it 'create instance' do
+    newalbum = MusicAlbum.new(publish_date: '2-21-2023', on_spotify: true)
+    expect(newalbum).to be_instance_of(MusicAlbum)
+  end
+
+  describe '#can_be_archived?' do
+    context 'on Spotify and publish date < 10 years' do
+      newalbum = MusicAlbum.new(publish_date: '2-21-2023', on_spotify: true)
+      it 'returns false' do
+        result = newalbum.send(:can_be_archived?)
+        expect(result).to be false
+      end
+    end
+
+    context 'on Spotify and publish date > 10 years' do
+      newalbum = MusicAlbum.new(publish_date: '2-21-2012', on_spotify: true)
+      it 'returns true' do
+        result = newalbum.send(:can_be_archived?)
+        expect(result).to be true
```

Comment:
You did a great job testing a private method using `: send` 👏 . However,  this approach has disadvantages majorly it breaks encapsulation 🚒  and it makes tests harder to refactor 🪨 . I kindly recommend that you implement a public method in your `MusicAlbum Class` to test your private method. This approach will help you to make your code and tests easier to manage and force you to design your code better 😃

Reply:
Ok, thanks for the suggestion, and for the approval.